DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs

```diff
@@ -0,0 +1,156 @@
+﻿using DashboardFrontend.Charts;
+using DashboardFrontend.ViewModels;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace DashboardFrontend.DetachedWindows
+{
+    /// <summary>
+    /// Interaction logic for ManagerListDetached.xaml
+    /// </summary>
+    public partial class ManagerListDetached : Window
+    {
+        public ManagerViewModel Vm { get; set; }
+        public ObservableCollection<ManagerWrapper> Managers { get; set; } = new();
+
+        public ManagerListDetached(ManagerViewModel managerViewModel) //Conversion parameter
+        {
+            Vm = managerViewModel;
+
+            InitializeComponent();
+
+            DataContext = Vm;
+        }
+
+        /// <summary>
+        /// Adds managers to the from the manager overview by calling the manager mover method for every manager selected.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void AddManager_Click(object sender, RoutedEventArgs e)
+        {
+            foreach (ManagerWrapper manager in datagridManagers.SelectedItems)
+            {
+                if (!Vm.WrappedManagers.Any(e => e.Manager.ContextId == manager.Manager.ContextId))
+                {
+                    DatagridManagerMover("Add", manager);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Removes managers to the from the manager details overview and charts by calling the manager mover method for every manager selected.

```

Comment:
to the from the

Reply:
lav en sugestion istedet for bare at kommentere det like diz:
```suggestion
        /// Removes managers from the manager details overview and charts by calling the manager mover method for every manager selected.

```